prince-npm.js

```diff
@@ -204,6 +205,23 @@ var downloadData = function (url) {
     });
 };
 
+/*  extract a zipfile (*.zip)  */
+var extractZipfile = function (zipfile, destdir) {
+    return new promise(function (resolve, reject) {
+        var zip = new streamzip({ file: zipfile });
+        zip.on("ready", function () {
+            zip.extract("prince-14.2-macos", destdir, function (error) {
```

Comment:
Unfortunately, the `node-stream-zip` library (and a few others I looked at) does not appear to have the equivalent of the `tar` library's `strip` option, so I've had to hard-code the parent directory name for the extraction to work as expected.

If you have a better approach, I'm all ears. Otherwise, this will just have to be updated whenever `princeDownloadURL` is.